javatests/arcs/sdk/wasm/CollectionApiTest.kt

```diff
@@ -13,6 +13,13 @@ package arcs.sdk.wasm
 
 class CollectionApiTest : AbstractCollectionApiTest() {
     private var stored = CollectionApiTest_OutHandle()
+    var x = 0.0;
+
+    init{
+        handles.ioHandle.onUpdate{ _ ->
```

Comment:
Style nit: this can go on the same line, and naming the argument is optional: `handles.ioHandle.onUpdate { x++ }`